capa/capabilities/common.py

```diff
@@ -52,8 +52,7 @@ def has_file_limitation(rules: RuleSet, capabilities: MatchResults, is_standalon
             logger.warning(" %s", line)
         logger.warning(" Identified via rule: %s", file_limitation_rule.name)
         if is_standalone:
-            logger.warning(" ")
-            logger.warning(" Use -v or -vv if you really want to see the capabilities identified by capa.")
+            pass
```

Comment:
`pass` here _for now_, because I think `is_standalone` is used in `capa_ghidra.py`.

Reply:
recommend adding a TODO so there's an issue to remind you to come back 😃